docs/software.md

```diff
@@ -64,6 +64,14 @@
 
 - [**Folding@Home - Use processing power to assist with disease research**](../software/computational_science/#foldinghome)
 
+## [Databases & Data stores](../software/database/#overview)
```

Comment:
I suggest to use `databases` (plural) for the URL/file name, like is it with `desktops`, `dns_servers` etc.

Reply:
This was my first intent, but I've reversed the naming, once I've checked the current notation. Almost everything is at singular [desktop/#chromium](https://dietpi.com/docs/software/desktop/#chromium), [remote_desktop/#tigervnc-server](https://dietpi.com/docs/software/remote_desktop/#tigervnc-server), [cloud/#owncloud](https://dietpi.com/docs/software/cloud/#owncloud) etc. There are only 3 exceptions: dns_servers, file_servers, hardware_projects. 

Singular is a good naming convention and we may continue using it. It is also often used in the industry.

Let me know what you think.

Reply:
> Almost everything is at singular

Ah true for `/desktop/` in the URL, while the category is named "Desktops", an accidental inconsistency 🤔.

In all other "singular" cases, the category name is indeed just an area where the contained software belongs to, like "Social", "Cloud", "Media" etc, where plural would sound wrong. In case of "... Servers", "... Clients" "Desktops" and "Databases", the plural forms are quite commonly used to mean the different available software/systems available.

For database servers "Data Storage" would be a singular form where I wouldn't use plural, but would be confused with what we have in "Cloud". I think "Databases" is very common and well understood.
__________
Another low-prio question: Shall we use "Data Stores" or "Data stores" or "data stores"? We have no strict convention for that 🤔. First word capital letter + first word after `&`, or only first word, or all words? In headings and listings I think there is also no general convention in languages, websites or such.

Reply:
Ok. I'll change the file name to `/databases/`, and keep the category name in capital words: _Databases & Data Stores_.